websites/M/MultiplayerPiano.net/metadata.json

```
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "224864947480297472",
+		"name": "antokek"
+	},
+	"service": "MultiplayerPiano.net",
```

```suggestion
	"service": "Multiplayer Piano",
```
Same with folder name.